tensorrt_llm/_torch/models/modeling_llama.py

```
@@ -309,32 +309,27 @@ def __init__(
 
     def forward(
         self,
         hidden_states: torch.Tensor,
         all_rank_num_tokens=None,
-        all_rank_max_num_tokens=None,
         final_all_reduce_params: Optional[AllReduceParams] = None,
         cutlass_min_latency_mode: Optional[bool] = False,
     ) -> torch.Tensor:
         # Only enable multi-stream for cuda graph since switch stream has extra host overhead
         # This design is mainly for low latency use case. Need to improve for max throughput use case.
         fn0 = lambda: self.shared_expert(hidden_states)
```

Ruff (0.12.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `fn0` as a `def`

(E731)